eclair-core/src/test/scala/fr/acinq/eclair/channel/states/ChannelStateTestsHelperMethods.scala

```diff
@@ -39,15 +39,15 @@ import fr.acinq.eclair.router.Router.ChannelHop
 import fr.acinq.eclair.transactions.Transactions
 import fr.acinq.eclair.transactions.Transactions._
 import fr.acinq.eclair.wire.protocol._
-import org.scalatest.{FixtureTestSuite, ParallelTestExecution}
+import org.scalatest.FixtureTestSuite
 
 import java.util.UUID
 import scala.concurrent.duration._
 
 /**
  * Created by PM on 23/08/2016.
  */
-trait ChannelStateTestsBase extends ChannelStateTestsHelperMethods with FixtureTestSuite with ParallelTestExecution {
+trait ChannelStateTestsBase extends ChannelStateTestsHelperMethods with FixtureTestSuite {
```

Comment:
Good idea, but it's really worth explaining what we're doing here. Can you add a comment here detailing why this does *not* extend `ParallelTestExecution` and recommending child test classes to contain a small-ish amount of tests (and the trick of dividing them into chunks as you did for normal and closing)?

Reply:
Went ahead and did that with c272f5446 and ca7a3ba81.